code/iaas/ha/src/main/java/io/cattle/platform/ha/monitor/impl/PingInstancesMonitorImpl.java

```diff
@@ -344,17 +354,20 @@ protected ReportedInstances getInstances(Ping ping) {
     }
 
     protected AgentAndHost loadAgentAndHostData(ImmutablePair<Long, String> agentIdAndHostUuid) {
```

Comment:
Minor: there is no real need to use Pair here, as you can just pass agentId/hostUUID as 2 separate params (and in this case you can avoid creating a new object + conversion back to long/string in loadAgentAndHostData).

I usually use pair only when its necessary (for return type for example).